src/DynamoManipulation/NodeManipulator.cs

```diff
@@ -65,9 +66,9 @@ public abstract class NodeManipulator : INodeManipulator
         /// <summary>
         /// Returns list of Gizmos used by this manipulator.
         /// </summary>
-        /// <param name="createIfNone">Whether to create new gizmo if not already present.</param>
+        /// <param name="createOrUpdate">Whether to create new gizmo or to update an existing one.</param>
         /// <returns>List of Gizmos.</returns>
-        protected abstract IEnumerable<IGizmo> GetGizmos(bool createIfNone);
+        protected abstract IEnumerable<IGizmo> GetGizmos(bool createOrUpdate);
```

Comment:
createOrUpdate?? means we are telling derived class to reuse existing gizmo. We need to explicitly mention that expectation here in the documentation.